ingestion_tools/scripts/data_validation/tests/annotation/helper_point.py

```diff
@@ -0,0 +1,30 @@
+from typing import Dict, List
+
+
+def point_count_consistent(
+    annotations: Dict[str, List[Dict]],
+    annotation_metadata: Dict[str, Dict],
+    annotation_file_to_metadata_file: Dict[str, str],
+):
+    """Check that the number of annotations is consistent between the metadata and the ndjson file."""
+
+    for annotation_filename, annotation_data in annotations.items():
+        print(f"Annotation Object: {annotation_filename}")
+        metadata_file = annotation_file_to_metadata_file[annotation_filename]
```

Comment:
print statements are an easy way to give more context when a pytest fails, but we can improve this logging in the future